cmd/anonymizer/app/anonymizer/anonymizer.go

```diff
@@ -65,14 +65,18 @@ type Anonymizer struct {
 
 // New creates new Anonymizer. The mappingFile stores the mapping from original to
 // obfuscated strings, in case later investigations require looking at the original traces.
-func New(mappingFile string, logger *zap.Logger) *Anonymizer {
+func New(mappingFile string, logger *zap.Logger, hashStandardTags, hashCustomTags, hashLogs, hashProcess bool) *Anonymizer {
```

Comment:
I would recommend defining a struct like AnonymizerOptions and moving the hashXXX params into fields of that struct. This way you can reuse it in many places (including copying it as is into Anonymized.options field.